tests/run/cpp_nested_names.pyx

```diff
@@ -0,0 +1,11 @@
+# tag: cpp
```

Comment:
Can we put this in tests/run/cpp_nested_classes? Adding a new test adds a little more overhead than adding to a test